task.sql

```diff
@@ -38,3 +38,16 @@ CREATE TABLE ProductInventory (
     PRIMARY KEY (ID)
 );
 --rollback DROP TABLE ProductInventory;
+
+--changeset mate.acamemy:5 labels:0.0.2
+CREATE TABLE Users (
+    ID INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
+    FirstName VARCHAR(50),
+    LastName VARCHAR(50),
+    Email VARCHAR(50)
+);
+--rollback DROP TABLE Users;
+
+--changeset mate.acamemy:6 labels:0.0.3
+CREATE INDEX Email ON Users(Email);
+--rollback ALTER TABLE Users DROP INDEX Email;
```

Comment:
![image](https://github.com/user-attachments/assets/8817b530-9ae0-4362-849a-91ea964cf6e7)
Add Eof(End-of-file), need to add a blank line at the end